.ci/release.sh

```diff
@@ -6,8 +6,11 @@ cp .travis.settings.xml $HOME/.m2/settings.xml && mvn deploy
 # Update version by 1
 newVersion=${TRAVIS_TAG%.*}.$((${TRAVIS_TAG##*.} + 1))
 
-# Replace first occurrence of TRAVIS_TAG with newVersion appended with SNAPSHOT 
-sed -i "0,/<version>$TRAVIS_TAG/s//<version>$newVersion-SNAPSHOT/" pom.xml
+# Update project version to next snapshot version
+mvn versions:set -DnewVersion=$newVersion-SNAPSHOT -DgenerateBackupPoms=false
 
-git commit pom.xml -m "Upgrade version to $newVersion-SNAPSHOT" --author "Github Bot <githubbot@gluonhq.com>"
+# Update Substrate to next snapshot version
+mvn versions:use-next-snapshots -Dincludes=com.gluonhq:substrate
```

Comment:
this generates a backup, you can add `-DgenerateBackupPoms=false` as well?

Reply:
Fixed!